bin/CoreData/Shaders/GLSL/v2/PBRTerrainBlend.glsl

```diff
@@ -1 +1,5 @@
-#error Not implemented
+#ifndef PBR
+    #define PBR
+#endif
+
+#include "M_LegacyTerrain.glsl"
```

Comment:
Adding PBR define to M_LegacyTerrain doesn't make it PBR :trollface:
But I don't care enough to demand it fixed.

Reply:
I didn't look too deep into it to be honest. At least it wasn't failing to load. I can fix the pbr-ness too.